android/cli/commands/_build.js

```diff
@@ -4208,7 +4174,8 @@ AndroidBuilder.prototype.runDexer = function runDexer(next) {
 		}),
 		injarsCore = [
 			this.buildBinClassesDir,
-			path.join(this.platformPath, 'lib', 'titanium-verify.jar')
+			path.join(this.platformPath, 'lib', 'titanium-verify.jar'),
```

Comment:
extract constants for these files paths since you're re-building them in multiple places.